packages/dataviews/src/style.scss

```diff
@@ -1,22 +1,26 @@
 .dataviews-wrapper {
 	width: 100%;
-	height: calc(100% - #{$grid-unit-40} * 2);
+	height: 100%;
 	overflow: auto;
-	padding: $grid-unit-40 $grid-unit-40 0;
```

Comment:
Since we're removing the padding here, probably we want to add some padding when we have no results and when showing the loading message. Besides the padding though, it would be good to improve them too - better messages or spinner/progress bar etc..